Framework/Core/src/DeviceSpecHelpers.cxx

```diff
@@ -945,6 +946,7 @@ boost::program_options::options_description DeviceSpecHelpers::getForwardedDevic
   // - child-driver is not a FairMQ device option but used per device to start to process
   bpo::options_description forwardedDeviceOptions;
   forwardedDeviceOptions.add_options()                                                                          //
+    ("severity", bpo::value<std::string>()->default_value("severity"), "severity level of the log")             //
```

Comment:
there should be `default_value("info")` here. But after I fixed this, in case the workflow was called with some device option it produces errors 
````
o2-tpc-reco-workflow --tpc-digit-reader '--infile tpcdigits.root' --input-type digits --output-type clusters,tracks --tracker-options "cont refX=83 bz=-5.0068597793" --tpc-track-writer "--treename events --track-branch-name Tracks --trackmc-branch-name TracksMCTruth"
...
[INFO] Starting tpc-clusterer on pid 28061
[ERROR] Unable to pass configuration to children
[INFO] Starting tpc-cluster-decoder on pid 28062
[ERROR] Unable to pass configuration to children
[INFO] Starting tpc-native-cluster-writer on pid 28097
[ERROR] Unable to pass configuration to children
[INFO] Starting tpc-tracker on pid 28098
[ERROR] Unable to pass configuration to children
[INFO] Starting tpc-track-writer on pid 28131
[ERROR] Unable to pass configuration to children
[INFO] Redeployment of configuration done.
...
[28035:tpc-digit-reader]: Error: option '--severity' cannot be specified more than once
[28061:tpc-clusterer]: [ERROR] error while setting up workflow: Error while parsing serialised workflow
[28061:tpc-clusterer]: [INFO] Process 28061 is exiting.
[28062:tpc-cluster-decoder]: [ERROR] error while setting up workflow: Error while parsing serialised workflow
[28062:tpc-cluster-decoder]: [INFO] Process 28062 is exiting.
[28097:tpc-native-cluster-writer]: [ERROR] error while setting up workflow: Error while parsing serialised workflow
[28097:tpc-native-cluster-writer]: [INFO] Process 28097 is exiting.
[28098:tpc-tracker]: [ERROR] error while setting up workflow: Error while parsing serialised workflow
[28098:tpc-tracker]: [INFO] Process 28098 is exiting.
...
28131:tpc-track-writer]: [ERROR] error while setting up workflow: Error while parsing serialised workflow
[28131:tpc-track-writer]: [INFO] Process 28131 is exiting.
````